nautobot_ssot/utils.py

```diff
@@ -0,0 +1,4 @@
+"""Utility functions for Nautobot SSOT App."""
+import logging
+
+logger = logging.getLogger("nautobot.ssot")
```

Comment:
I don't quite understand why this change is necessary?

Also:
```suggestion
"""Utility functions for Nautobot SSoT App."""
import logging

logger = logging.getLogger("nautobot.ssot")
```